### PR TITLE
Implementing LoginComplete Service

### PR DESCRIPTION
**Pull request by umututku03**
I have implemented the LogInComplete Use-Case, allowing registered users in our database to access and utilize all the services offered by our application.

## Motivation and Context

To utilize the full range of services and use-cases our app offers, users are required to log in. Consequently, it is essential to develop both the complete login use-case and its corresponding CA Engine components.

## Your Changes

I have successfully implemented all the components, including interface adapters and more, in alignment with the principles of Clean Architecture and SOLID.

**Type of change** (select all that apply):

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Refactoring (internal change to codebase, without changing functionality)
- [ ] Test update (change that modifies or updates tests only)
- [ ] Documentation update (change that modifies or updates documentation only)
- [ ] Other (please specify):

## Checklist

- [X] I have performed a self-review of my own code.
- [ ] I have added tests for my changes. 
- [ ] Coverage tests
- [x] Coding style (conforms to Google Java Style Guide)
- [x] Especially follows section 7 (Javadoc is present for every public class, public or protected member)
- [x] Especially follows section 4.6 (Multiple consecutive blank lines are not encouraged)
- [X] Correctness (follows CA and SOLID, is a reasonable and not overly complicated solution, etc.)
- [X] Description of pull request is clear, consise, and reasonably detailed

### CODE REVIEW

File: src/services/login_complete/LoginCompletePresenter.java
Comment (bryjjch) [inline src/services/login_complete/LoginCompletePresenter.java:1]: It would be wise to do something similar to loginstates/signupstates.

**Review by bryjjch — CHANGES_REQUESTED** [review]
Need to figure out setting errors.

**Review by HussainE4 — CHANGES_REQUESTED** [review]
There a few Java Style issues that should be fixed; the UserDataAccessObject should implement the LoginCompleteUserDataAccessInterface and method names should be altered. Also, review the message regarding States

**Review by HussainE4 — APPROVED** [review]
works

**Review by skrukwa — APPROVED** [review]
lgtm

**Pull request merged**